video/probe.go

```diff
@@ -20,7 +20,22 @@ type Prober interface {
 
 type Probe struct{}
 
-func (p Probe) ProbeFile(url string, ffProbeOptions ...string) (iv InputVideo, err error) {
+func (p Probe) ProbeFile(url string, ffProbeOptions ...string) (InputVideo, error) {
+	iv, err := p.runProbe(url, ffProbeOptions...)
+	if err == nil {
+		return iv, nil
+	}
+
+	// ignore these probing errors if found and re-run with fatal loglevel to obtain the probe data
+	errMsg := strings.ToLower(err.Error())
+	if strings.Contains(errMsg, "parametric stereo signaled to be not-present but was found in the bitstream") ||
```

Comment:
How about creating a const with all error messages to ignore? Like this:

```golang
var IGNORE_ERR_MESSAGES = []string{
	"parametric stereo signaled to be not-present but was found in the bitstream",
	"non-existing pps 0 referenced",
}

...

	for _, ignErr := range IGNORE_ERR_MESSAGES {
		if strings.Contains(errMsg, ignErr) {
                        return p.runProbe(url, "-loglevel", "fatal")
		}
	}
```

Reply:
Looks good, I'll pick this up since Max is out until next week

Reply:
Done